components/NavBar/NavBar.js

```diff
@@ -48,12 +56,35 @@ export const navItems = [
     link: URL_INV_CART,
   },
 ];
+export const actionItems = [
+  {
+    title: 'Add new item',
+    icon: <AddIcon key={3} style={{ marginRight: 5 }} />,
```

Comment:
```suggestion
    icon: <AddIcon style={{ marginRight: 5 }} />,
```

can help me remove all the key attributes for `actionItems` and `navItems` too not sure why i added them before, but there doesn't seem to be a need for it.